automated_packaging/prepare_changelog.pl

```diff
@@ -0,0 +1,60 @@
+#!usr/bin/perl
+$CODE_PATH=$ARGV[0];
```

Comment:
Why do we need to get the path from user? I always assumed we will install this scripts and then call them from project's own folder or the script will clone the repo.